paddle/fluid/framework/operator.h

```diff
@@ -297,6 +299,64 @@ template <>
 std::vector<Tensor*> ExecutionContext::MultiOutput<Tensor>(
     const std::string& name) const;
 
+class RuntimeInferShapeContext : public InferShapeContext {
```

Comment:
This is exposed just because ngraph_operator is not like other paddle operators? Can we avoid this by making ngraph_operator much like a paddle operator?

Reply:
Let me look into this.